src/conv-core/memory-lock.C

```diff
@@ -13,72 +7,72 @@ static void meta_init(char **argv)
 void *meta_malloc(size_t size)
 {
   void *result;
-  MEM_LOCK_AROUND( result = mm_malloc(size); )
+  MEM_LOCK_AROUND( result = mm_impl_malloc(size); )
```

Comment:
Does `MEM_LOCK_AROUND` still work with the macro definition removed?

Reply:
Yes, it is in memory.C now.